packages/extras/src/lib/audio/PositionalAudio/PositionalAudio.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts">
+  import { PositionalAudio as ThreePositionalAudio } from 'three'
+  import { Three, useThrelteAudio } from '@threlte/core'
+  import type { PositionalAudioProps } from './PositionalAudio.svelte'
+
+  type $$Props = PositionalAudioProps
+
+  export let position: $$Props['position'] = undefined
+  export let scale: $$Props['scale'] = undefined
+  export let rotation: $$Props['rotation'] = undefined
+  export let viewportAware: $$Props['viewportAware'] = false
+  export let inViewport: $$Props['inViewport'] = false
+  export let castShadow: $$Props['castShadow'] = undefined
+  export let receiveShadow: $$Props['receiveShadow'] = undefined
+  export let frustumCulled: $$Props['frustumCulled'] = undefined
+  export let renderOrder: $$Props['renderOrder'] = undefined
+  export let visible: $$Props['visible'] = undefined
+  export let userData: $$Props['userData'] = undefined
+  export let dispose: $$Props['dispose'] = undefined
+
+  export let autoplay: $$Props['autoplay'] = undefined
+  export let detune: $$Props['detune'] = undefined
+  export let source: $$Props['source'] = undefined
+  export let volume: $$Props['volume'] = undefined
+  export let loop: $$Props['loop'] = undefined
+  export let filters: $$Props['filters'] = undefined
+  export let playbackRate: $$Props['playbackRate'] = undefined
+
+  export let play: $$Props['play'] = undefined
+  export let pause: $$Props['pause'] = undefined
+  export let stop: $$Props['stop'] = undefined
+
+  export let id: $$Props['id'] = undefined
+  export let directionalCone: $$Props['directionalCone'] = undefined
+  export let refDistance: $$Props['refDistance'] = undefined
+  export let rolloffFactor: $$Props['rolloffFactor'] = undefined
+  export let distanceModel: $$Props['distanceModel'] = undefined
+  export let maxDistance: $$Props['maxDistance'] = undefined
+
+  const { getAudioListener } = useThrelteAudio()
+
+  const listener = getAudioListener(id)
+
+  if (!listener) {
+    throw new Error(`No Audiolistener with id ${id} found.`)
+  }
+
+  export const audio = new ThreePositionalAudio(listener)
+
+  $: {
+    if (refDistance !== undefined) audio.setRefDistance(refDistance)
+    if (rolloffFactor !== undefined) audio.setRolloffFactor(rolloffFactor)
+    if (distanceModel !== undefined) audio.setDistanceModel(distanceModel)
+    if (maxDistance !== undefined) audio.setMaxDistance(maxDistance)
+    if (directionalCone !== undefined) {
+      audio.setDirectionalCone(
+        directionalCone.coneInnerAngle,
+        directionalCone.coneOuterAngle,
+        directionalCone.coneOuterGain
+      )
+    }
+  }
+</script>
+
+<Three
+  type={audio}
+  {position}
```

Comment:
Same here, please remove `{position}` et al and insert these via `{...$$restProps}` and remove these props from the script block